rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -80,6 +83,9 @@
   // default values
   private static final int DEFAULT_BLOCK_SIZE = 1 * 1024 * 1024; // 1MB
   private static final int DEFAULT_CLIENT_TIMEOUT = 10000; // ms
+  private static final String DEFAULT_CACHE_METADATA_FILE_SUFFIX = "_mdfile";
+  private static final int DEFAULT_CACHE_VALIDATION_INITIAL_DELAY = 30000; // ms
```

Comment:
Is cache validation every 30sec very frequent ? Can we something like 5 min or 10 min ? @vrajat

Reply:
@abhishekdas99 @vrajat My bad here. I think we had discussed 30min, but my mind was still stuck on seconds from the other intervals.

Updated the interval & initial delay to 30min